Aptoide/appcoins-wallet-android
# [APPC-1058] Include wallet creation on purchase flow

**What does this PR do?**

   Adds the flow of automatically creating the wallet when the user tries to do an IaP, has the wallet installed, but has no wallet yet.
  This PR fixed a bug when creating a wallet and the user would close the application when the wallet was being created. This would incapacitate the user of doing an IaP

**Where should the reviewer start?**

OneStepPaymentReceiver.java
Erc681Receiver.java

**How should this be manually tested?**

1.In case APPC-1059 still hasn’t gone to dev, go to EthereumNetworkRepository.java and change line 52 from 0 to 4
2.Clear app data from wallet (to erase any wallets), go to trivial drive and buy gas and confirm that the loading screen appears and after the wallet is created it shows the payments dialog
3.Do the same as above, but this time to buy anti freeze
4.Press buy (either one) again and check if it doesn’t show the loading screen
Repeat the second step but this time press back when the loading screen appears. Now press buy again and it should appear the payment dialog. 

**What are the relevant tickets?**

https://aptoide.atlassian.net/browse/APPC-1058
